### ISSUE TITLE
Guests are connected but can't see and hear each other

### ISSUE THREAD

**Issue by jaakj** [jitsi/jitsi-meet#2826]
I have a following issue: conference guests (including Moderator) can connect to room, their mic and camera are turned on but all the other incoming guests seem to muted their mics and turn off the cameras. And the case is the same regardless from which side to look - my mic and camera are always on but my guests are muted (even though they actually aren't) and vice versa:
![image](https://user-images.githubusercontent.com/38441695/38834164-462c2102-41d0-11e8-9683-87d9e34c0e5a.png)

Maybe it wasn't the clearest explanation but I hope you get the point :) Can you show me a direction where to start looking?

**Comment by damencho** [comment]
I suppose this is on your deployment? 
It looks to me like you are missing this step: https://github.com/jitsi/jitsi-meet/blob/master/doc/quick-install.md#advanced-configuration

**Comment by jaakj** [comment]
Thanks for quick reply! Yes, this is my own deployment. 
I added two lines into /etc/jitsi/videobridge/sip-communicator.properties:
org.ice4j.ice.harvest.NAT_HARVESTER_LOCAL_ADDRESS='my-private-ip'
org.ice4j.ice.harvest.NAT_HARVESTER_PUBLIC_ADDRESS='my-public-ip'

Not sure about the ports though - ufw status shows that all the necessary ports (tcp/443, tcp/4443, udp/10000) are open but port checking web tools indicate that my 4443 and 10000 ports seem to be closed. Can this be the root of the problems?

**Comment by c526537207** [comment]
I think the server bandwidth is too low.The videobridge is unable to transfer streams.

**Comment by jaakj** [comment]
I'm using DigitalOcean standard droplet. Even though they claim that the bandwidth limit is 1TB/month, the last info says that they are still not charging for (extra) bandwidth. So I guess the bandwidth may not be a problem.

**Comment by c526537207** [comment]
I tested it with 1M of bandwidth, and I had the same problem.[https://jitsi.org/jitsi-videobridge-performance-evaluation/](url),The study found that each user had to allocate 515k of bandwidth.

**Comment by saghul** [comment]
> Not sure about the ports though - ufw status shows that all the necessary ports (tcp/443, tcp/4443, udp/10000) are open but port checking web tools indicate that my 4443 and 10000 ports seem to be closed. Can this be the root of the problems?

Yes, those ports need to be open for the media to be able to flow. Specially 10000/udp, since that's the default for audio / video media.

**Comment by c526537207** [comment]
@saghul It seems to be the question, how do I turn on the 10000 / udp, where is it configured?

**Comment by damencho** [comment]
@c526537207  This is the default port and does not need configurations. 

@jaakj DigitalOcean does not need the advanced configuration as their machines are directly configured with the public address. 
So I'm not sure what values have you added as 'my-private-ip' and 'my-public-ip'. I think you can remove them restart jvb and jicofo and try again.
And yes if the ports are blocked you will have the exact same experience as on the screenshot.

**Comment by c526537207** [comment]
@jaakj Has the problem been solved?

**Comment by c526537207** [comment]
@damencho @saghul I have already tested, 10000 / udp, 443 / TCP, 4443 / TCP, the port has been opened.But there are still problems with the subject.

**Comment by damencho** [comment]
@c526537207 Can you send me a URL to test?

**Comment by c526537207** [comment]
udp test command nc -vuz ip port @damencho

**Comment by damencho** [comment]
I'm asking for URL to your deployment, to take a look by entering 3 participants in a conference.

**Comment by c526537207** [comment]
@damencho https://sp.weishichuang.cn/

**Comment by c526537207** [comment]
@damencho I've configured it this way, but it's still wrong.I didn't use Let's Encrypt, but the certificate of other institutions. Does this matter?

**Comment by c526537207** [comment]
@damencho Where did you see it? I only configured the public network address.

**Comment by damencho** [comment]
No, the certificate does not matter. 
You need to configure both public and private address in /etc/jitsi/videobridge/sip-communicator.properties then restart jvb and also make sure you forward the ports to the machine.
Open in chrome this address: chrome://webrtc-internals, then open three tabs with same conference URL, then in the webrtc-internal you will see entries for different sessions, in those that are using the bridge under setRemoteDescription I see the ip address that had been offered.

**Comment by marclaporte** [comment]
I have been getting this on https://meet.jit.si/ in last few months. So both participants think the other person did not show up.

It's rare but it happens. Workaround: leave and come back (but you need to know about and remember that this bug can happen)
